controller/place.js

```diff
@@ -16,7 +16,7 @@ function setup( backend ){
     var query = req.clean.ids.map( function(id) {
       return {
         _index: 'pelias',
-        _type: id.type,
+        type: id.types,
```

Comment:
Ok, looking at [this question on the ES discourse page](https://discuss.elastic.co/t/should-filtering-by-type-and-type-specific-search-url-return-different-results/7008), it looks like the difference is that `_type` is used by many wrappers (like the JS one we use) to change the actual URL, which can obviously only filter against one type. `type`, meanwhile, actually adds type to part of the query, and Elasticsearch will accept an array of possible values to match against no problem.